scripts/mergehex.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+from intelhex import IntelHex
+
+import argparse
+import os
+
+
+def merge_hex_files(output, input_hex_files):
+    ih = IntelHex()
+
+    for hex_file_path in input_hex_files:
+        if not os.path.isfile(hex_file_path):
+            raise RuntimeError("%s is not a valid file path." % hex_file_path)
```

Comment:
What do you think about using argparse.FileType to avoid having to write this validation code?

https://stackoverflow.com/a/35736815

Reply:
Note that Python 3.4 is a dependency.

`find_package(PythonInterp 3.4)
`

Reply:
- [ ] Use FileType